cvat-ui/src/components/task-page/task-page.tsx

```diff
@@ -37,19 +37,26 @@ function TaskPageComponent(): JSX.Element {
         deletes,
         updates,
         jobsFetching,
+        bulkFetching,
     } = useSelector((state: CombinedState) => ({
         deletes: state.tasks.activities.deletes,
         updates: state.tasks.activities.updates,
         jobsFetching: state.jobs.fetching,
+        bulkFetching: state.selection.fetching,
     }), shallowEqual);
-    const isTaskUpdating = updates[id] || jobsFetching;
+    const isTaskUpdating = (updates[id] || jobsFetching) && !bulkFetching;
 
     const receiveTask = (): Promise<Task[]> => {
         if (Number.isInteger(id)) {
             const promise = core.tasks.get({ id });
             promise.then(([task]: Task[]) => {
                 if (task) {
                     setTaskInstance(task);
+                    if (task.jobs && Array.isArray(task.jobs)) {
+                        const jobsList = [...task.jobs] as JobsList;
```

Comment:
task.jobs already have specified type: `Job[]` and it is not compatible with JobsList having also count. So, this cast does not seem to be correct. 

Maybe to rewrite it just as:

```
dispatch(jobsActions.getJobsSuccess(
    Object.defineProperty([...task.jobs], 'count', { value: ... })
));
```